platform/osx/os_osx.mm

```diff
@@ -927,20 +927,29 @@ static void keyboardLayoutChanged(CFNotificationCenterRef center, void *observer
 	keyboard_layout_dirty = true;
 }
 
+static bool displays_arragment_dirty = true;
+static void displaysArrangmentChanged(CGDirectDisplayID displayID, CGDisplayChangeSummaryFlags flags, void *userInfo) {
```

Comment:
Typo, and should use snake_case and not camelCase, so `displays_arrangement_changed`.